build-config/make/busybox.make

```diff
@@ -13,17 +14,17 @@
 # This is a makefile fragment that defines the build of busybox
 #
 
-BUSYBOX_VERSION		= 1.20.2
+BUSYBOX_VERSION		= 1.25.1
```

Comment:
kind of weird the spaces in the version.  even the old line shows spaces.  strange.